src/modules/layerManager/components/LayerManager.js

```diff
@@ -1,8 +1,8 @@
 import { html } from 'lit-html';
-import { BaElement } from '../../../BaElement';
-import { $injector } from '../../../../injection';
+import { BaElement } from '../../BaElement';
+import { $injector } from '../../../injection';
 import { repeat } from 'lit-html/directives/repeat.js';
-import { modifyLayer } from './../../../../store/layers/layers.action';
+import { modifyLayer } from './../../../store/layers/layers.action';
 import css from './layerManager.css';
 
 /**
```

Comment:
Can we precise the description here to something like "Renders a list of layers representing their order on a map and provides actions like reordering, removing and changing visibility and opacity"